index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="css/stylelist.css">
+	<title>Document</title>
+</head>
+<body>
+
+	<div class="content"> <!-- основное поле типа боди -->
+		<div class="header"> <!-- чердак -->
+			<div class="icon"></div>
+			<div class="spoilmeny"><a href="https://www.kinopoisk.ru/series/468051/" target="blank"><img src="image/menu.png" width="40" height="40" alt="hi"></a></div>
+			<div class="fuckingimg">Картинки</div>
+			<div class="post">Почта</div>
+		</div>
+
+		<div class="mainbodyof"> <!-- святая середина контента мать его -->
+			<!-- говнище1 -->
+			<div class="someimage"><a href="https://icdn.lenta.ru/images/2020/03/02/04/20200302041157409/square_320_e82dddbfb6f968b5a11887e55bb2b5d3.png"  target="blank"><img src="image/Без названия.png"  alt="none"></a>
+			</div>
+			<!-- говнище2 -->
+			<div class="someinput">
```

Comment:
Если у тебя составное имя, то используй следующий синтаксис

1. someinput => some_input

Для Джс было бы так

2. someinput => someInput

Reply:
Еще важно

Старайся не использовать обобщенные имена
```suggestion
			<div class="email_input">
```